.gitmodules

```diff
@@ -70,3 +70,6 @@
 [submodule "vendor/github.com/pkg/profile"]
 	path = vendor/github.com/pkg/profile
 	url = https://github.com/pkg/profile
+[submodule "vendor/github.com/j-keck/arping"]
```

Comment:
The `arping` project does not have any tests :-(

Will weave integration tests fail if the arping does not work as expected?